pkg/printer/printer.go

```diff
@@ -95,6 +97,14 @@ func (p *Printer) printK8sValidationWarning(warning Warning) {
 	fmt.Fprintln(out)
 }
 
+func (p *Printer) printK8sValidationWarning(warning Warning) {
+	p.printInColor("[?] Kubernetes schema validation\n", p.Theme.Colors.White)
```

Comment:
There is a problem with printing white on white background using this function, talk to @TzlilSwimmer123 she has already found a solution for it.